modin/pandas/test/test_series.py

```diff
@@ -721,13 +721,17 @@ def test_aggregate_numeric_except(request, data, func):
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 def test_aggregate_error_checking(data):
-    modin_series, _ = create_test_series(data)  # noqa: F841
+    modin_series, pandas_series = create_test_series(data)
 
+    assert pandas_series.aggregate("ndim") == 1
     assert modin_series.aggregate("ndim") == 1
     with pytest.warns(UserWarning):
```

Comment:
Does this mean that `UserWarning` should be appeared for function call on `modin_series` and `pandas_series`?
What if the warning appears for only one of the two series objects?

Reply:
The warning should be appeared for `modin_series` only.

Reply:
I can imagine a situation where a warning will appear only for `pandas_series` object and the check will pass, but this is not what we expect.

Reply:
Can you show an example?

Reply:
@anmyachev is right, it's better to check for warnings throwing explicitly for modin Series, changed this test at [the last commit](https://github.com/modin-project/modin/pull/3209/commits/de68a8acafcfbf226a910a7ce95a11c9d8ec4df1)

Reply:
@dchigarev , okay but I would like to look at the example anyway.